### PR TITLE
feat(compute): add sample and test for local SSD disk

### PR DESCRIPTION
**Pull request by Sita04**
Sample to create disk with local SSD

### CODE REVIEW

File: compute/createInstanceWithLocalSSD.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  //    boot disk of an instance.
+  //
+  //    sourceImage: source image to use when creating this disk.
+  //    You must have read access to this disk. This can be one of the publicly available images
+  //    or an image from one of your projects.
+  //    This value uses the following format: "projects/{project_name}/global/images/{image_name}"
+  //
+  //    autoDelete: boolean flag indicating whether this disk should be deleted
+  //    with the VM that uses it.
+  async function createDiskFromImage(
```
Comment (alexander-fenster) [inline compute/createInstanceWithLocalSSD.js:88]: Why would this and other functions be `async`? They don't have anything asynchronous in them, so they should be synchronous and used without `await`.

Comment (Sita04) [reply compute/createInstanceWithLocalSSD.js]: Thanks for the comment.

I've removed async from a couple of struct type functions. The other function to create an instance does require async.

**Comment by pattishin** [comment]
Hi @Sita04! When you get a chance, do you mind checking out the compute test failure? Will get this merged in once that is resolved.

**Review by pattishin — APPROVED** [review]
Awesome, lgtm! :shipit:

**Pull request merged**